### ISSUE TITLE
Swicth audio output when connected to bluetooth headphones

### ISSUE THREAD

**Issue by madeindjs** [SimpleMobileTools/Simple-Dialer#96]
When I answer to a call with a connected bluetooth device, audio output is automatically redirected to my headphone. But I may be able to switch to audio device without disconnect my headphones.

**Comment by shanemd** [comment]
I am also interested in this feature. I have seen other dialer apps use the 'speakerphone' icon to do this switching by turning it into a selector icon when bluetooth is connected. When pressed, it presents the options of 'phone', 'speaker', or 'bluetooth', for where the audio will go.

The use case and motivaiton that I have for this is: I have a bluetooth module in my car to play audio through my car speakers. This module does not have a usable microphone though, so when I answer a call while in my car, I have to manually turn bluetooth off before the other party can hear me talking. It would actually be nice to be able to set a default behaviour for this selector, and have the audio default to the 'phone' even when bluetooth is connected.

**Comment by Eorlariel** [comment]
I would be interested, too.

**Comment by nvkv** [comment]
Definitely would be nice to have this

**Comment by czadikem** [comment]
Yes this would be great.  I know Koler has this might want to look at that since it is open source too.

**Comment by cicciobellohalabua2** [comment]
this is what happens in koler, https://github.com/Chooloo/koler could you take a look at the source code to add this function?

https://user-images.githubusercontent.com/71934037/148682445-92913b84-9295-4c16-b778-8e018a8af015.mp4

**Comment by MusKaya** [comment]
+follow

**Comment by J-Schaefer** [comment]
Is this feature actually still open? I think this has been implemented, right?
